lib/elixir_sense/plugins/ecto/query.ex

```diff
@@ -167,7 +167,7 @@ defmodule ElixirSense.Plugins.Ecto.Query do
 
     relations =
       Enum.map_join(associations, ", ", fn assoc ->
```

Comment:
```suggestion
      Enum.map_join(associations, ", ", fn
        %{related: related, related_key: related_key} ->
          "`#{inspect(related)} (#{inspect(related_key)})`"
        %{related: related} ->
          # Ecto.Association.ManyToMany does not define :related_key
          "`#{inspect(related)}`"
```

Reply:
without that change the string would end with not very helpful ` (nil)`

Reply:
Great call, thanks for adding that!